### DISCUSSION TITLE
Show status of fork & allow sync fork

### DISCUSSION THREAD

**Discussion by Unfocused** [community/community#50358] [Mobile] [answered]
### Select Topic Area

Bug

### Body

When you have a fork of a repository that is one or more commits behind (or ahead of) the upstream branch, the web UI on github.com will:

- Show the number of commits behind & ahead of the upstream branch
- If the branch is behind the upstream branch, it'll show a "Sync fork" dropdown. Opening that will show:
  - A button to compare this branch with the upstream branch 
  - A button to update the branch from the upstream branch 

However, the GitHub mobile app only shows that a repository is a fork, and what it's a fork of. It doesn't:
- Show the status of the fork
- Allow comparing with upstream
- Allow syncing the fork

As a manual workaround, you can of course switch back & forth between fork & upstream to manually take note of recent commit history of each branch to see if they differ. Although realistically that's only going to work in simple cases.

Practical effects:
- You can view a fork and have no idea that it's (for example) months out of date
- You can view a fork and have no idea how it differs from upstream
- You can edit, commit, and create a PR from a fork. But you'd have no idea how behind upstream the fork is, so you wouldn't know if you edit is still relevant or correct
- You can edit, commit, and create a PR from a fork.  But you can't sync the fork with upstream, so your PR could have significant merge conflicts
- With a short amount of time to do some quick housekeeping tasks on your phone, you can't go through your forks and quickly sync them from upstream

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the feedback @Unfocused. We added your feedback to our internal tracking issue and hope to have something for this soon.
